tests/views/test_accounts.py

```diff
@@ -16,16 +16,18 @@
 with open(f"{project_root}/test_data/uaa/user_by_id.json") as json_data:
     uaa_user_by_id_json = json.load(json_data)
 
-test_email = "fake@ons.gov.uk"
 user_id = "fe2dc842-b3b3-4647-8317-858dab82ab94"
+user_email = user_id
```

Comment:
Sorry I missed this one.  The user email should be an actual email, not an id.   If you revert the change back to `test_email = "fake@ons.gov.uk"` then it should be fine.